frame/assets/src/tests.rs

```diff
@@ -999,3 +999,30 @@ fn querying_roles_should_work() {
 		assert_eq!(Assets::freezer(0), Some(4));
 	});
 }
+
+#[test]
+fn normal_asset_create_and_destroy_callbacks_should_work() {
+	new_test_ext().execute_with(|| {
+		assert!(sp_io::storage::get(b"asset_created").is_none());
```

Comment:
I would put `use sp_io::storage;` on the top and simply call `storage::get()`

Reply:
Sure, changed.